plugins-root/t/check_icmp.t

```diff
@@ -35,51 +35,72 @@ my $hostname_invalid   = getTestParameter( "NP_HOSTNAME_INVALID",
 
 my $res;
 
+# 1
```

Comment:
what's the idea of these comments?

Reply:
To bee honest, just to help me with counting them

Reply:
I would just run the test once after adding test cases. It then tells you the new number of tests and it can be changed.

Reply:
Removed the comments